### PR TITLE
{tools}[GCCcore/9.3.0] Subversion v1.14.0

### PR DESCRIPTION
**Pull request by zao**
(created using `eb --new-pr`)
Bump version of Subversion and its dependencies for 2020a.

### CODE REVIEW

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
freja - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (skylake), Python 3.8.5
See https://gist.github.com/5ea7f1eb7897add767c7316b88ee871f for a full test report.

File: easybuild/easyconfigs/s/Serf/Serf-1.3.9-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+dependencies = [
+    ('APR', '1.7.0'),
+    ('APR-util', '1.6.1'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it is nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.1p'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```
Comment (Micket) [inline easybuild/easyconfigs/s/Serf/Serf-1.3.9-GCCcore-9.3.0.eb:29]:
```suggestion
osdependencies = [OS_PKG_OPENSSL_DEV]
```

**Review by Micket — CHANGES_REQUESTED** [review]
https all the things!

**Comment by zao** [comment]
Two of the Apache mirrors (EU/US) have invalid certs, should they remain?

**Comment by Micket** [comment]
Only use https when they are valid. The test-suite should be smart enough to check that

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4f865ea8457c4b4a154565a40b9a430b for a full test report.

**Comment by Micket** [comment]
Going in, thanks @zao!

**Pull request merged**